unicodetools/data/uca/4.1.0/CollationTest/CollationTest_NON_IGNORABLE.txt

```diff
@@ -1437,7 +1437,7 @@ FE13 0062;	# (︓) PRESENTATION FORM FOR VERTICAL COLON	[023E 0F85 | 0020 0020 |
 0017 0021;	# ('') <END OF TRANSMISSION BLOCK>	[0253 | 0020 | 0002 |]
 0018 0021;	# ('') <CANCEL>	[0253 | 0020 | 0002 |]
 0019 0021;	# ('') <END OF MEDIUM>	[0253 | 0020 | 0002 |]
-001A 0021;	# ('') <SUBSTITUTE>	[0253 | 0020 | 0002 |]
+001A 0021;	# ('') <SUBSTITUTE>	[0253 | 0020 | 0002 |]
```

Comment:
FYI the change in this line and others like it below is to change from having no actual character between the single quotes ('') to having that actual 0x1A SUBSTITUTE character (invisible) between the quotes (''). Seems like it was likely an intentional change in the source (not an artifact of reconstruction), but is in a comment anyway.

Reply:
I know. U+001A is "end of file" on DOS/Windows, and I guess these characters disappear when you are on a Windows machine. I had started with files from Laurentiu, and there were so many changes that stuff like this has likely snuck in. I will watch out for this in the next snapshot.

See issue #2.